src/busco/busco/script.sh

```diff
@@ -0,0 +1,74 @@
+[[ "$par_opt_out_run_stats" == "false" ]] && unset par_opt_out_run_stats
+[[ "$par_quiet" == "false" ]] && unset par_quiet
+[[ "$par_restart" == "false" ]] && unset par_restart
+[[ "$par_auto_lineage" == "false" ]] && unset par_auto_lineage
+[[ "$par_auto_lineage_euk" == "false" ]] && unset par_auto_lineage_euk
+[[ "$par_auto_lineage_prok" == "false" ]] && unset par_auto_lineage_prok
+[[ "$par_augustus" == "false" ]] && unset par_augustus
+[[ "$par_long" == "false" ]] && unset par_long
+[[ "$par_scaffold_composition" == "false" ]] && unset par_scaffold_composition
+[[ "$par_miniprot" == "false" ]] && unset par_miniprot
+
+
+tmp_dir=$(mktemp -d -p "$meta_temp_dir" busco_XXXXXXXXX)
+prefix=$(openssl rand -hex 8)
+
+busco \
+    --in "$par_input" \
+    --mode "$par_mode" \
+    --out "$prefix" \
+    --out_path "$tmp_dir" \
+    ${meta_cpus:+--cpu "${meta_cpus}"} \
+    ${par_lineage_dataset:+--lineage_dataset "$par_lineage_dataset"} \
+    ${par_augustus:+--augustus} \
+    ${par_augustus_parameters:+--augustus_parameters "$par_augustus_parameters"} \
+    ${par_augustus_species:+--augustus_species "$par_augustus_species"} \
+    ${par_auto_lineage:+--auto-lineage} \
+    ${par_auto_lineage_euk:+--auto-lineage-euk} \
+    ${par_auto_lineage_prok:+--auto-lineage-prok} \
+    ${par_contig_break:+--contig_break $par_contig_break} \
+    ${par_datasets_version:+--datasets_version "$par_datasets_version"} \
+    ${par_e_value:+--evalue "$par_e_value"} \
+    ${par_force:+--force} \
+    ${par_limit:+--limit "$par_limit"} \
+    ${par_long:+--long} \
+    ${par_metaeuk_parameters:+--metaeuk_parameters "$par_metaeuk_parameters"} \
+    ${par_metaeuk_rerun_parameters:+--metaeuk_rerun_parameters "$par_metaeuk_rerun_parameters"} \
+    ${par_miniprot:+--miniprot} \
+    ${par_offline:+--offline} \
+    ${par_opt_out_run_stats:+--opt-out-run-stats} \
+    ${par_quiet:+--quiet} \
+    ${par_restart:+--restart} \
+    ${par_scaffold_composition:+--scaffold_composition} \
+    ${par_tar:+--tar} \
+    ${par_download_base_url:+--download_base_url "$par_download_base_url"} \
+    ${par_download_path:+--download_path "$par_download_path"} \
+    ${par_download:+--download "$par_download"} 
+
+
+
+if [[ -n "$par_short_summary_json" ]]; then
+    find "$tmp_dir/$prefix" -maxdepth 1 -type d -name 'run_*' -exec cp {}/short_summary.json "$par_short_summary_json" \;
+fi
+if [[ -n "$par_short_summary_txt" ]]; then
+    find "$tmp_dir/$prefix" -maxdepth 1 -type d -name 'run_*' -exec cp {}/short_summary.txt "$par_short_summary_txt" \;
+fi
+if [[ -n "$par_full_table" ]]; then
+    find "$tmp_dir/$prefix" -maxdepth 1 -type d -name 'run_*' -exec cp {}/full_table.tsv "$par_full_table" \;
+fi
+if [[ -n "$par_missing_busco_list" ]]; then
+    find "$tmp_dir/$prefix" -maxdepth 1 -type d -name 'run_*' -exec cp {}/missing_busco_list.tsv "$par_missing_busco_list" \;
+fi
+if [[ -n "$par_output_dir" ]]; then
+    cp -r $tmp_dir/$prefix/run_* $par_output_dir
+fi
```

Comment:
Maybe to avoid doing all of these repeated `find` statements, how about the following?

```suggestion
out_dir=$(find "$tmp_dir/$prefix" -maxdepth 1 -name 'run_*')

if [[ -n "$par_short_summary_json" ]]; then
    cp "$out_dir/short_summary.json" "$par_short_summary_json"
fi
if [[ -n "$par_short_summary_txt" ]]; then
    cp "$out_dir/short_summary.txt" "$par_short_summary_txt"
fi
if [[ -n "$par_full_table" ]]; then
    cp "$out_dir/full_table.tsv" "$par_full_table"
fi
if [[ -n "$par_missing_busco_list" ]]; then
    cp "$out_dir/missing_busco_list.tsv" "$par_missing_busco_list"
fi
if [[ -n "$par_output_dir" ]]; then
    if [[ -d "$par_output_dir" ]]; then
        rm -r "$par_output_dir"
    fi
    cp -r "$out_dir" "$par_output_dir"
fi
```

Note that I removed the `-type d` because when `--auto_lineage` is set, the `$dir/$prefix/run_*` directory is a symlink:

```
# ls -l output/1053cb13b3d3779b/
total 12
drwxr-xr-x. 1 root root  108 Jan 30 19:26 auto_lineage
drwxr-xr-x. 1 root root  134 Jan 30 19:27 logs
lrwxrwxrwx. 1 root root   32 Jan 30 19:27 run_eukaryota_odb10 -> auto_lineage/run_eukaryota_odb10
-rw-r--r--. 1 root root 1775 Jan 30 19:27 short_summary.specific.eukaryota_odb10.1053cb13b3d3779b.json
-rw-r--r--. 1 root root  622 Jan 30 19:27 short_summary.specific.eukaryota_odb10.1053cb13b3d3779b.txt
```